src/commcare_cloud/ansible/roles/commcarehq/tasks/celery.yml

```diff
@@ -16,7 +16,6 @@
       - "{{supervisor_celery_workers}}"
 
 - name: Add celery_bash_runner files
-  become: yes
```

Comment:
Just curious are these removals just cleanup because sudo isn't necessary?

Reply:
All tasks had/needed it, so I just put `become: true` in src/commcare_cloud/ansible/deploy_commcarehq.yml when including it instead, as it was done for similar cases. It should be safe because it's not included anywhere else.